distributions/standalone/assembly-whitelist.txt

```diff
@@ -256,6 +258,33 @@
 /mule-standalone-${productVersion}/lib/opt/uuid-3.4.0.jar
 /mule-standalone-${productVersion}/lib/opt/xmlbeans-2.6.0.jar
 
+## RAML BEGIN parser dependencies
+/mule-standalone-${productVersion}/lib/opt/annotations-3.0.0.jar
+/mule-standalone-${productVersion}/lib/opt/btf-1.2.jar
+/mule-standalone-${productVersion}/lib/opt/commons-lang-2.6.jar
+/mule-standalone-${productVersion}/lib/opt/jackson-annotations-2.4.0.jar
+/mule-standalone-${productVersion}/lib/opt/jackson-core-2.4.4.jar
+/mule-standalone-${productVersion}/lib/opt/jackson-coreutils-1.8.jar
+/mule-standalone-${productVersion}/lib/opt/jackson-databind-2.4.4.jar
+/mule-standalone-${productVersion}/lib/opt/jackson-module-jsonSchema-2.4.4.jar
+/mule-standalone-${productVersion}/lib/opt/javax.json-1.0.4.jar
+/mule-standalone-${productVersion}/lib/opt/javax.json-api-1.0.jar
+/mule-standalone-${productVersion}/lib/opt/jopt-simple-4.6.jar
+/mule-standalone-${productVersion}/lib/opt/json-schema-core-1.2.5.jar
+/mule-standalone-${productVersion}/lib/opt/json-schema-validator-2.2.6.jar
+/mule-standalone-${productVersion}/lib/opt/jsr305-3.0.0.jar
+/mule-standalone-${productVersion}/lib/opt/juniversalchardet-1.0.3.jar
+/mule-standalone-${productVersion}/lib/opt/libphonenumber-6.2.jar
+/mule-standalone-${productVersion}/lib/opt/mailapi-1.4.3.jar
+/mule-standalone-${productVersion}/lib/opt/msg-simple-1.1.jar
+/mule-standalone-${productVersion}/lib/opt/raml-parser-2-1.0.11.jar
+/mule-standalone-${productVersion}/lib/opt/rhino-1.7R4.jar
+/mule-standalone-${productVersion}/lib/opt/snakeyaml-1.15.jar
+/mule-standalone-${productVersion}/lib/opt/uri-template-0.9.jar
+/mule-standalone-${productVersion}/lib/opt/xmlschema-core-2.2.1.jar
+/mule-standalone-${productVersion}/lib/opt/yagi-1.0.11.jar
+## RAML END parser dependencies
```

Comment:
The distributions module was now moved to mule-distributions project and these changes should be done there